src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/FileIO/FileSystem.Windows.vb

```diff
@@ -0,0 +1,187 @@
+﻿Imports System
+Imports System.ComponentModel
+Imports System.Diagnostics
+Imports Microsoft.VisualBasic.CompilerServices
+Namespace Microsoft.VisualBasic.FileIO
+    Partial Public Class FileSystem
```

Comment:
>FileSystem [](start = 25, length = 10)

Could this file be merged back into FileSystem.vb, with #If blocks for the platform-specific regions?

Reply:
I could merge FileSystem.vb and FileSystem.Portable.vb but given how much stuff has been removed and moved around I not sure it would help with reviews. I would need to stich them back together 1 function at a time but I can do next week if required. I would have actually liked to further split the files into related parts (I don't know why TextFieldParser is even in here) to make them more manageable.